### ISSUE TITLE
Add some kind of indicator

### ISSUE THREAD

**Issue by danieldiekmeier** [jorinvo/github-backup#12]
Currently, it does its thing and then finished.

Maybe add some kind of output what it's currently doing? Could be optional behind a flag.

**Comment by jorinvo** [comment]
There is a `-verbose` flag. 
It could be a bit more talkative though.

**Comment by danieldiekmeier** [comment]
I think it would be more common to call it `--verbose` and `-V`/`-v`.

For example, `python --help` yields

```
-v     : verbose (trace import statements); also PYTHONVERBOSE=x
         can be supplied multiple times to increase verbosity
-V     : print the Python version number and exit (also --version)
```

and `node --help` yields

```
  -v, --version         print Node.js version
  -e, --eval script     evaluate script
  -p, --print           evaluate script and print result
```

suggesting that words should always have `--` as a prefix.

**Comment by jorinvo** [comment]
Yes, right now we're using the default Go args package. 
For Go tools the conventions is to have only one dash and a long name.

There are other good args libraries for Go that support other ways of specifying args. 
(For example: https://github.com/spf13/pflag, https://github.com/spf13/cobra)
Could think about switching to make it less confusing.

Right now it's nice that this simple tools has no dependencies outside the standard library.

**Comment by jorinvo** [comment]
But we should definitely update the usage / help message to also list the `-verbose` option.

**Comment by danieldiekmeier** [comment]
Oh, I didn't know the convention was different.

I also just tested `-verbose`, and it does exactly what I want, so this can be closed when the messages are updated.

**Comment by jorinvo** [comment]
Maybe we can fix this by implementing #11.

**Comment by jorinvo** [comment]
But I also think it's a valid point that we should have better messages when using `-verbose`. I think we can improve this.
